build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2056,6 +2056,17 @@ message CacheCapabilities {
   // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
   // requests.
   repeated Compressor.Value supported_batch_update_compressors = 7;
+
+  // Maximum size a single CAS blob can have in the remote cache system.
```

Comment:
This sentence contradicts the last sentence in this block. How about:
```suggestion
  // The maximum size that the server will accept for CAS blob uploads.
```

Reply:
Thanks!